.circleci/config.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         type: string
     environment:
       CI_E2E_FILENAME: "f99e7b0c/rel-nightly"
+      CHANNEL: nightly
```

Comment:
should this be pinned to `stable`. do we have a stable test artifact?

Reply:
I think you're right, we should pin to `stable`.

Reply:
missed this one?
```suggestion
      CHANNEL: stable
```